Assets/FlowEnt/Scripts/Runtime/Core/Abstract/AbstractAnimation/AbstractAnimation.cs

```diff
@@ -180,8 +180,9 @@ protected override void StopInternal(bool triggerOnCompleted)
             switch (playState)
             {
                 case PlayState.Building:
-                case PlayState.Finished:
                     break;
+                case PlayState.Finished:
```

Comment:
can you update the animations tests to reflect the fact that this issue was fixed? (Calling Stop twice)

Reply:
As in write a new test for it?

Reply:
call stop on assert again and check if it throws or not